.travis.yml

```diff
@@ -5,10 +5,14 @@ before_install:
   - sudo apt-get update
   - sudo apt-get install libevent-dev daemontools
 install:
+  - pip install -U pip wheel
   - pip install -r requirements-dev.txt
 before_script:
   - flake8 djangopeople
   - flake8 tests
 script:
   - psql -c 'CREATE DATABASE djangopeople;' -U postgres
   - python manage.py test
+cache:
```

Comment:
According to https://docs.travis-ci.com/user/caching/, it looks like `cache: pip` is enough.